group-coordinator/src/main/resources/common/message/ConsumerGroupMetadataValue.json

```diff
@@ -21,6 +21,8 @@
   "flexibleVersions": "0+",
   "fields": [
     { "name": "Epoch", "versions": "0+", "type": "int32",
-      "about": "The group epoch." }
+      "about": "The group epoch." },
+    { "name":  "SubscriptionMetadataHash", "versions":  "0+", "type":  "int64",
+      "about":  "The hash of subscription metadata." }
```

Comment:
Unfortunately, we cannot add a new field like this because it breaks the backward compatibility. Instead, we should use a tagged field.